src/components/dashboard/Dashboard.jsx

```diff
@@ -521,6 +524,30 @@ const Dashboard = props => {
     }
   }, [isCitizen])
 
+  useEffect(async () => {
+    const dontShowAgain = await AsyncStorage.getItem('dontShowWelcomeOffer')
+    if (!dontShowAgain) {
+      return
+    }
+
+    const country = await fetch('https://get.geojs.io/v1/ip/country.json')
+      .then(_ => _.json())
+      .then(_ => _.country)
+
+    const isEligible = supportedCountries.split(',').includes(country)
+
+    if (isWeb && welcomeOfferActive && isEligible) {
+      fireEvent('migration_invited')
+
+      showDialog({
+        content: <WelcomeOffer onDismiss={hideDialog} />,
+        titleStyle: { paddingTop: 0, marginTop: 0, minHeight: 'auto' },
+        onDismiss: hideDialog,
```

Comment:
if user dismiss dialog, maybe there should be some kind of delay until we show it again.
otherwise everytime user goes to dashboard he will see the popup

Reply:
Added a check where it shows now only once per day